scene/gui/slider.cpp

```diff
@@ -325,6 +370,7 @@ void Slider::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "scrollable"), "set_scrollable", "is_scrollable");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "tick_count", PROPERTY_HINT_RANGE, "0,4096,1"), "set_ticks", "get_ticks");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "ticks_on_borders"), "set_ticks_on_borders", "get_ticks_on_borders");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "find_ticks"), "set_find_ticks", "is_finding_ticks");
```

Comment:
I think that the name is not clear, and neither is the description for that matter. It could be something like `ticks_from_steps`. Setting this value should also probably make the property for the fixed number of ticks readonly (you would need to use `_validate_property`).